bin/pasta_form_patchwork_relations.py

```diff
@@ -0,0 +1,62 @@
+import sys
+import os
+
+from logging import getLogger
+
+sys.path.append(os.path.abspath(os.path.join(os.path.dirname(__file__), '..')))
+from pypasta import *
+
+log = getLogger(__name__[-15:])
+
+
+def form_patchwork_relations(config, prog, argv):
+    parser = argparse.ArgumentParser(prog=prog,
+                                     description='Form patch groups using Patchwork IDs instead of Message IDs')
+    parser.add_argument('-infile', metavar='infile', default=None,
+                        help='The patch groups file to be used as input'
+                             'If not specified will default to value of PATCH_GROUPS config variable')
+
+    args = parser.parse_args(argv)
+
+    if config.mode != config.Mode.MBOX:
+        log.error('Only works in Mbox mode!')
+        return -1
+
+    config.repo.register_mbox(config)
+    mbox = config.repo.mbox
+
+    if args.infile is None:
```

Comment:
If args.infile takes a sane default value, then this is superfluous.

Reply:
yes, makes sense